test/hw_mkl.f90

```diff
@@ -0,0 +1,13 @@
+program hello
```

Comment:
It's good to include math ops in the test program. If we include mkl by default, we can just roll this into the existing fortran test program/action.

Reply:
Yes, but that would also require installing BLAS/LAPACK for the gnu compilers - for now I would leave it separate, and make the optional math library install the default version once there is something for all compilers/OS.